warc_metadata_sidecar.py

```diff
@@ -125,7 +127,7 @@ def determine_soft404(bytes_payload):
 def create_warcinfo_payload(new_file, operator=None, publisher=None):
     """Collect WARC fields to create warcinfo record payload."""
     hostname = socket.gethostname()
-    version = pkg_resources.require('warc-metadata-sidecar')[0].version
+    version = __version__
     warc_doc = 'http://bibnum.bnf.fr/WARC/WARC_ISO_28500_version1_latestdraft.pdf'
     warcinfo_payload = {'software': 'warc-metadata-sidecar/' + version,
```

Comment:
We could use `__version__` directly here instead of having another variable called `version`.